test/parallel/test-console-write-without-ignore-errors.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common');
+const { Console } = require('console');
+const { Writable } = require('stream');
+const assert = require('assert');
+const ignoreErrors = false;
+const expected = 'foobarbaz';
```

Comment:
Could you either scope this to the first test, or name it something else. This isn't actually "expected" output in the second test.